python/py-xraylarch/Portfile

```diff
@@ -54,25 +52,34 @@ if {${name} ne ${subport}} {
 
     depends_run-append \
                     port:py${python.version}-asteval \
+                    port:py${python.version}-charset-normalizer \
+                    port:py${python.version}-dill \
                     port:py${python.version}-h5py \
+                    port:py${python.version}-hdf5plugin \
                     port:py${python.version}-fabio \
+                    port:py${python.version}-imageio \
                     port:py${python.version}-importlib-metadata \
                     port:py${python.version}-lmfit \
                     port:py${python.version}-matplotlib \
                     port:py${python.version}-numdifftools \
-                    port:py${python.version}-Pillow \
+                    port:py${python.version}-numpy \
```

Comment:
`numpy` is already listed as `depends_lib` which implies at build- and runtime, so please remove from here